frontend/src/components/phones/onboarding/RealPhoneSetup.tsx

```diff
@@ -0,0 +1,318 @@
+      className={`${styles["step-input-verificiation-code-timeout"]} ${styles["is-hidden"]} `}
+    >
+      <p>{l10n.getString("phone-onboarding-step3-error-exipred")}</p>
+      <Button className={styles.button} type="submit">
+        {l10n.getString("phone-onboarding-step3-error-cta")}
+      </Button>
+    </div>
+  );
+
+  const codeEntryPlaceholder = "000000";
+
+  return (
+    <div
+      className={`${styles.step}  ${styles["step-input-verificiation-code"]} `}
+    >
+      <div ref={leadRef} className={styles.lead}>
+        <div
+          className={`${styles["step-input-verificiation-code-lead-default"]}`}
+        >
+          {/* Default state */}
+          <img src={EnteryVerifyCode.src} alt="" width={300} />
+          <h2>{l10n.getString("phone-onboarding-step2-headline")}</h2>
+        </div>
+        <div
+          className={`${styles["step-input-verificiation-code-lead-error"]} `}
+        >
+          {/* Timeout error state */}
+          <img src={EnteryVerifyCodeError.src} alt="" width={170} />
+          <h2 className={`${styles["is-error"]} `}>
+            {l10n.getString("phone-onboarding-step3-code-fail-title")}
+          </h2>
+          <p>{l10n.getString("phone-onboarding-step3-code-fail-body")}</p>
+        </div>
+      </div>
+      {/* Add error class of `mzp-is-error` */}
+
+      {/* TODO: Add logic to display timeout error */}
+      {errorTimeExpired}
+
+      {/* <form onSubmit={onSubmit} className={`${styles.form}`}> */}
+      <form ref={formRef} onSubmit={onSubmit} className={styles.form}>
+        {/* TODO: Make remaining_time count backwards with "X minutes, XX seconds" */}
+        <Localized
+          id="phone-onboarding-step3-body"
+          vars={{
+            phone_number: formatPhone(
+              phoneWithMostRecentlySentVerificationCode.number
+            ),
+            remaining_time: formatTime(remainingTime),
+          }}
+          elems={{
+            span: <span className={`${styles["phone-number"]}`} />,
+            strong: <strong />,
+          }}
+        >
+          <p />
+        </Localized>
+
+        <input
+          placeholder={codeEntryPlaceholder}
+          required={true}
+          onChange={onChange}
+          ref={inputRef}
+          autoFocus={true}
+        />
+        {/* TODO: Add logic to show success/fail on submit */}
+        <Button className={styles.button} type="submit">
+          {l10n.getString("phone-onboarding-step3-button-cta")}
+        </Button>
+
+        {/* TODO: Enable logic to "go back" to previous step */}
+        <Button
+          onClick={() => props.onGoBack()}
+          className={styles.button}
+          type="button"
+          variant="secondary"
+        >
+          {l10n.getString("phone-onboarding-step3-button-edit")}
+        </Button>
+      </form>
+
+      {/* TODO: Resubmit phone number for verification and reset count down */}
+      {/* onClick={() => props.requestPhoneVerification(phoneNumber)} */}
```

Comment:
suggestion (non-blocking): remove old(?) commented code?

Reply:
Removed in de725280b1a1fa9cb499491b2078e30f3e43dcd1